pkg/lib/page.scss

```diff
@@ -189,3 +189,8 @@ html:not(.index-page) body {
 .ct-icon-times-circle {
     color: var(--pf-global--danger-color--100);
 }
+
+.ct-page-add-top-bottom-padding.pf-c-page__main-section.pf-m-no-padding.pf-m-padding-on-sm {
+    padding-top: var(--pf-c-page__main-section--PaddingTop);
+    padding-bottom: var(--pf-c-page__main-section--PaddingBottom);
+}
```

Comment:
At this point, should we consider an opt-in ct-* class with a media query that just sets the padding?

```css
@media screen and (max-width: $pf-global--breakpoint--sm) {
  .ct-page-mobile-padding {
    padding: padding-top: var(--pf-c-page__main-section--PaddingTop) 0 var(--pf-c-page__main-section--PaddingBottom);
  }
}
```
(Which hardcodes the top and bottom, like what's in this PR at the moment.)

or

```css
@media screen and (max-width: $pf-global--breakpoint--sm) {
  .ct-page-mobile-padding {
    padding-left: 0;
    padding-right: 0;
  }
}
```
(Which just nullifies the left and right margins, leaving the top and bottom untouched.)

It would be less CSS and less React one className to add (`.ct-page-mobile-padding`).

Meanwhile, we could also ask PF to add a paddingWidth option (or something like that).